src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    Double getAreaCalculator();
```

Comment:
```suggestion
    Double getArea();
```

this method should return area, not AreaCalculator